File: src/js/code/jest/comment.test.js
const { qase } = require("jest-qase-reporter/jest");
// ANCHOR: syntax
// describe("Example: comment.test.js", () => {
//   test("A test case with qase.comment()", () => {
     qase.comment("Add a comment to the result");
     //// test logic here
//   });
// });
// ANCHOR_END: syntax

File: src/java/code/junit5/CommentTest.java
package com.example;

import io.qase.junit5.Qase;
import org.junit.jupiter.api.Test;
//ANCHOR:syntax
// public class CommentTest {
//     @Test
//     public void testWithComment_success() {
         Qase.comment("This shall appears under the 'actual result' field of the result");
//         //test logic here
// }
//ANCHOR_END:syntax